std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/DoubleStorageFacade.java

```diff
@@ -0,0 +1,92 @@
+package org.enso.table.data.column.storage.numeric;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.function.ToDoubleFunction;
+import org.enso.table.data.column.storage.*;
+import org.enso.table.data.column.storage.type.FloatType;
+import org.enso.table.data.column.storage.type.StorageType;
+
+/** A facade for a column storage that converts the stored type to a double. */
+public class DoubleStorageFacade<T> implements ColumnDoubleStorage {
```

Comment:
```suggestion
public final class DoubleStorageFacade<T> implements ColumnDoubleStorage {
```
unless you want clueless developers to subclass, make the class `final`.